src/auth/dto/authcreds.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsString, IsNotEmpty, IsEmail } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class AuthCredsDto {
+  @IsString()
+  @ApiProperty({ description: 'The email of the user' })
+  @IsEmail()
```

Comment:
PullProofAI

There is a redundant validation decorator `@IsString()` for the `email` field since `@IsEmail()` already validates it as a string. Remove the redundant decorator.
```diff
-  @IsString()
```


<!-- This is an auto-generated comment by PullProofAI -->